### PR TITLE
[ADS Web] Fix Plotly notebook rendering issue 

### LINKED ISSUE #15951: ADS Web - Plotly not rendering in Notebooks
Azure Data Studio - Insiders
Version: 1.31.0-insider
Commit: d476672921025539b505cbb86191f48ba4983919
Date: 2021-06-25T17:53:46.989Z (4 days ago)
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.106 Safari/537.36

Steps to Reproduce:

1. ADS Web Image and Notebooks 
2. Using either Plotly or KQLmagic to render a graph 

Error:

image

### PR DESCRIPTION
**Pull request by VasuBhog**
This PR fixes #15951, the Plotly rendering issue within ADS Web Notebooks was not rendering due to an uncaught promise when trying to locate: plotly.js-dist-min module. As such we need to include this to render Plotly graphs within ADS Web mode. 

Confirmed with latest ADS Web image in the build pipeline that it displays properly show below:
image

### CODE REVIEW

File: src/vs/code/browser/workbench/workbench-web-dev.html [hunk 1]
```diff
@@ -56,6 +56,7 @@
 			'tas-client-umd': `${baseUrl}/node_modules/tas-client-umd/lib/tas-client-umd.js`,
 			'iconv-lite-umd': `${baseUrl}/node_modules/iconv-lite-umd/lib/iconv-lite-umd.js`,
 			'jschardet': `${baseUrl}/node_modules/jschardet/dist/jschardet.min.js`,
+			'plotly.js-dist-min': `${window.location.origin}/static/node_modules/plotly.js-dist-min/plotly.min.js`
```
Comment (caohai) [inline src/vs/code/browser/workbench/workbench-web-dev.html:59]: `${baseUrl}/node_modules/plotly.js-dist-min/plotly.min.js` ?

**Review by caohai — COMMENTED** [review]
Let's wait for the docker img for verification.

**Comment by caohai** [comment]
LGTM. Can you create an issue for this and add screenshots either in the issue or in the PR?

**Review by caohai — APPROVED** [review]

**Comment by Charles-Gagnon** [comment]
Can you include a brief description of what the issue was and how this fixes it?

(also did you mean to link the issue you just created for this?)

**Comment by VasuBhog** [comment]
> Can you include a brief description of what the issue was and how this fixes it?
> 
> (also did you mean to link the issue you just created for this?)

Yup this fixes #15951. I have included a screenshot of the issue there and the fix for it here.

**Review by Charles-Gagnon — APPROVED** [review]
Was this error showing up in the console logs at least? It'd be good to make sure that missing package errors like this are easily spotted in case there's any others we're missing.

**Comment by VasuBhog** [comment]
> Was this error showing up in the console logs at least? It'd be good to make sure that missing package errors like this are easily spotted in case there are any others we're missing.

It was after some debugging that I was able to see it in the console, as it was not shown until I stepped through the component.

**Pull request merged**